lib/common.js

```diff
@@ -5,26 +5,26 @@ const expressBrowserify = require('express-browserify')
 const { makeTextureAtlas } = require('./atlas')
 const { prepareBlocksStates } = require('./models')
 
-function setupRoutes (app, version) {
+function setupRoutes (app, version, prefix) {
   app.use(compression())
 
-  app.use('/', express.static(path.join(__dirname, '../public')))
-  app.get('/index.js', expressBrowserify(path.join(__dirname, '../src/index.js')))
-  app.get('/worker.js', expressBrowserify(path.join(__dirname, '../src/worker.js')))
+  app.use(prefix + '/', express.static(path.join(__dirname, '../public')))
+  app.get(prefix + '/index.js', expressBrowserify(path.join(__dirname, '../src/index.js').replace('/* placeholder */', '+ ' + prefix)))
```

Comment:
instead of the placeholder hack you could use https://stackoverflow.com/questions/23480336/defining-global-variable-for-browserify